test/eulalie/test/lambda.cljc

```diff
@@ -0,0 +1,39 @@
+(ns eulalie.test.lambda
+  (:require [eulalie.lambda.util :as lu]
+            #? (:clj [clojure.test :refer [testing]]
+                :cljs [cljs.test :refer-macros [testing]])
+            [eulalie.test.common :refer [issue-raw! creds]]
+            [glossop.core #? (:clj :refer :cljs :refer-macros) [go-catching <?]]
+            [eulalie.test.common :as test.common
+             #? (:clj :refer :cljs :refer-macros) [deftest is]]
+            [eulalie.util :refer [env!]]))
+
+(def lambda-role-arn (env! "LAMBDA_ROLE_ARN"))
+(def function-zipped "UEsDBAoAAAAAAEBL+0iHxOHaUwAAAFMAAAAHAAAAdGVzdC5qc2V4cG9ydHMuaGFuZGxlciA9IChldmVudCwgY29udGV4dCwgY2FsbGJhY2spID0+IHsgY2FsbGJhY2sobnVsbCwgIkhlbGxvIHdvcmxkIik7IH07UEsBAhQACgAAAAAAQEv7SIfE4dpTAAAAUwAAAAcAAAAAAAAAAAAAAAAAAAAAAHRlc3QuanNQSwUGAAAAAAEAAQA1AAAAeAAAAAAA")
+(defn function-name [] (str "eulalie-test-" (rand-int 2147483647))) ;JVM Integer/MAX_VALUE
+
+(deftest ^:integration ^:aws create-test-function
+         (if (not-empty (:secret-key creds))
+           (let [function-name (function-name)]
+             (go-catching
+               (testing "Create function"
+                 (is (= 201 (get-in (<? (issue-raw! {:creds creds
```

Comment:
I don't know how these aren't failing on Travis, as `LAMBDA_ROLE_ARN` will be nil AFAICT